api/server/routes/assistants/actions.js

```
@@ -175,13 +190,15 @@ router.delete('/:assistant_id/:action_id/:model', async (req, res) => {
+    const assistantUpdateData = { actions };
+    if (!assistant_data) {
+      assistantUpdateData.user = req.user.id;
+    }
     promises.push(
       updateAssistantDoc(
-        { assistant_id },
-        {
-          actions: updatedActions,
-          user: req.user.id,
-        },
+      { assistant_id },
```

ESLint

[error] Expected indentation of 8 spaces but found 6.

(indent)